src/main/java/org/example/OccurrenceWordPoc.java

```diff
@@ -0,0 +1,56 @@
+package org.example;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class OccurrenceWordPoc {
+    static String inputFilePath;
+    static String searchWord;
+
+    /*
+    Entering the path
+    once if the input is received it shows processing
+    and executes if the file is present or not present
+     */
+    public static void main(String[] args) {
+        inputFilePath = args[0];
+        searchWord = args[1];
+        System.out.println(inputFilePath);
```

Comment:
Probably needs to handle cases where one or more arguments are not passed in

Reply:
Added a check for length of the arguments.